app/models/pregnancy.rb

```diff
@@ -54,4 +54,12 @@ def self.most_recent
     order('created_at DESC').limit(1).first
   end
 
+	def recent_calls
+		calls.order("created_at DESC").limit(10)
+	end
+
+	def old_calls
```

Comment:
I think we can optimize this a little with the postgres cmd / activerecord method `offset` -- something like `calls.order('created_at desc').offset(10)` should work fab for this!